jest.config.js

```diff
@@ -1,5 +1,6 @@
 /** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
   preset: 'ts-jest',
-  testEnvironment: 'node'
+  testEnvironment: 'node',
+  maxWorkers: 1
```

Comment:
can you add a comment here to say that this line is to force tests to run sequentially and then link to the issue you created.